src/lib.rs

```diff
@@ -321,7 +322,10 @@ impl Check {
 
         // Create a report for each crate.
         // We want to run all the checks, even if one returns `Err`.
-        let all_outcomes: Vec<anyhow::Result<(String, CrateReport)>> = match &self.current.source {
+        let all_outcomes: Vec<anyhow::Result<(String, Option<CrateReport>)>> = match &self
+            .current
+            .source
+        {
```

Comment:
Might your local setup have a custom rustfmt config or something? I'm surprised by this unrelated change.

Reply:
I don't think I have a custom setup

Reply:
Ah I missed that, my bad.